channels/rdpecam/client/encoding.c

```diff
@@ -96,27 +150,70 @@ static BOOL ecam_encoder_compress_h264(CameraDeviceStream* stream, const BYTE* s
 {
 	UINT32 dstSize = 0;
 	BYTE* srcSlice[4] = { 0 };
+	int srcLineSizes[4] = { 0 };
 	BYTE* yuv420pData[3] = { 0 };
 	UINT32 yuv420pStride[3] = { 0 };
-	prim_size_t size = { 0 };
-	size.width = stream->currMediaType.Width;
-	size.height = stream->currMediaType.Height;
+	prim_size_t size = { stream->currMediaType.Width, stream->currMediaType.Height };
```

Comment:
please `WINPR_ASSERT(stream);` before dereferencing

Reply:
if I'm correct static function, only called from this module => not needed, we control the caller